src/utils/utils-content-script.ts

```diff
@@ -1,9 +1,89 @@
+import { Message, MessageType } from '../static/sharedTypes';
+import {
+  aboutTheJobClass,
+  optimizeCvButtonTitle,
+  optimizeCvButtonStylesClass,
+  saveButtonClass,
+} from '../utils/constants';
+import { formatClass } from './utils-common';
+
 export function extractTextFromElement(elementClass: string): string {
   if (!elementClass || elementClass === '') return '';
 
-  const element = document.querySelector(elementClass);
+  const formattedElementClass = formatClass(elementClass);
+
+  const element = document.querySelector(formattedElementClass);
   if (!element) return '';
   let textContent = element.textContent || '';
   textContent = textContent.replace('About the job', '').trim();
   return textContent;
 }
+
+export function appendSibling(
+  targetElement: HTMLElement,
+  newElement: HTMLElement
+) {
+  // get parent element
+  const parent = targetElement.parentNode;
+
+  // insert new element after target element
+  parent.insertBefore(newElement, targetElement.nextSibling);
+}
+
+export function sendMessageToBackground(
+  message: string,
+  type: MessageType
+): void {
+  chrome.runtime.sendMessage({
+    message,
+    type,
+  });
+}
+
+export async function createButton() {
+  const saveButtonEl = document.querySelector(
```

Comment:
classes might not be enough, you should use a "css selector" (it's a string to define elements in the DOM) - https://www.w3schools.com/cssref/css_selectors.php

Reply:
also it might be any element not only "save button" so make it more generic